doc/factory/custom_vars.html

```diff
@@ -1847,6 +1873,20 @@ <h3><a name="dyn_vars"></a>Dynamically generated variables</h3>
             <td><b>CONDORG_SUBCLUSTER</b></td>
             <td>Int</td>
           </tr>
+          <tr>
+            <td><b>GLIDEIN_BLACKHOLE</b></td>
+            <td>Expr (Bool)</td>
+            <td>
+              <p>
+                The variable indicates if the glidein have been detected as a
+                blackhole based on the stats obtained from the STARTD. The stats
+                are always the same for all slots as it is a global statistic
+                about the whole startd. By default is False. More details in
+                blackhole detection section, on the bottom of who webpage
+                (Lifetime of a Glidein).
```

Comment:
True if the Glidein has been detected as a black hole, False otherwise. 
The black hole condition is evaluated using GLIDEIN_BLACKHOLE_NUMJOBS, GLIDEIN_BLACKHOLE_RATE, and HTCondor statistics about the jobs completing in the Glidein, and it affects the whole Glidein (all its static or partitionable slots).
See the <a href="#blackhole">blackhole detection section</a> below for more details.